ngraph/test/visitors/op/atan.cpp

```diff
@@ -0,0 +1,27 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "gtest/gtest.h"
+
+#include "ngraph/ngraph.hpp"
+#include "ngraph/op/util/attr_types.hpp"
+#include "ngraph/opsets/opset1.hpp"
+
+#include "util/visitor.hpp"
+
+using namespace std;
+using namespace ngraph;
+using ngraph::test::NodeBuilder;
+
+TEST(attributes, atan_op)
+{
+    NodeBuilder::get_ops().register_factory<op::Atan>();
```

Comment:
ould you please rebase your PR to the latest master. And please use parametrized visitor API test which was introduced in this PR: #6181

Reply:
fixed, changed to use parametrized visitor API test